libs/clients/icelandic-health-insurance/health-insurance/src/lib/clients-health-insurance-v2.module.ts

```diff
@@ -13,7 +13,6 @@ export class HealthInsuranceV2Client {
       fetchApi: createEnhancedFetch({
         name: 'clients-health-insurance',
         organizationSlug: 'sjukratryggingar',
-        treat400ResponsesAsErrors: true,
```

Comment:
This one is a little out of the ordinary as the health insurance api has some internal validation in the submit endpoint for the AccindentNotification application that returns a 412 on validation errors. So If we remove this the application can be moved to a success state without the actual application being submitted to Sjukratryggingar.

Reply:
Could that be handled by the client library integrating with the insurance api?

Reply:
@obmagnusson  4xx responses are still thrown even though this property is removed. The client generated code always throws for response code >= 400. This property was controlling log level and affecting the circuit breaker.